azure-kusto-ingest/azure/kusto/ingest/_ingestion_properties.py

```diff
@@ -21,6 +21,20 @@ class DataFormat(Enum):
     multijson = "multijson"
 
 
+class IngestionMappingType(Enum):
+    Csv = "Csv"
```

Comment:
we should fix this across, but python seems to favour upper case naming convention for enums 
https://docs.python.org/3/library/enum.html#creating-an-enum